universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -167,7 +167,9 @@ public void putRepeatedEntities(Path path, ImmutableList<String> entityNames) {
   }
 
   private void putNull(Path path) {
-    put(path, null);
+    if (!path.isArrayElement()) {
```

Comment:
Thoughts (blocking): would this make more sense to check ifNull at line 192 and only add the value if not null there? The thinking I have is that putNull is an indication that the user answers the question with an empty answer, v.s. the user did not answer the question.

Reply:
I think there may be valid uses for putting null in the applicant data, but for arrays we should have empty arrays instead.

Reply:
Yeah, I meant it might still be useful to insert an empty array `[]` here. The behavior of the code would be to do nothing, which doesn't distinguish between answering with an empty answer or not answering at all.